apps/allocations/contracts/Allocations.sol

```diff
@@ -520,26 +545,7 @@ contract Allocations is AragonApp {
 
     function _canMakePayment(uint64 _accountId, uint256 _amount) internal view returns (bool) {
         Account storage account = accounts[_accountId];
-        return _getRemainingBudget(_accountId) >= _amount && vault.balance(account.token) >= _amount && _amount > 0;
-    }
-
-    function _getRemainingBudget(uint64 _accountId) internal view returns (uint256) {
-        Account storage account = accounts[_accountId];
-        if (!account.hasBudget) {
-            return MAX_UINT256;
-        }
-
-        uint256 budget = account.budget;
-        uint256 spent = periods[_currentPeriodId()].accountStatement[_accountId].expenses[account.token];
-
-        // A budget decrease can cause the spent amount to be greater than period budget
-        // If so, return 0 to not allow more spending during period
-        if (spent >= budget) {
-            return 0;
-        }
-
-        // We're already protected from the overflow above
-        return budget - spent;
+        return getRemainingBudget(_accountId) >= _amount && vault.balance(account.token) >= _amount && _amount > 0;
```

Comment:
We removed the `_` from the beginning. Why's that?

Reply:
It's a public function now

Reply:
ah wait I'll just make a public wrapper around this private function. Thanks for calling this out